turbinia/workers/bulk_extractor.py

```diff
@@ -73,14 +74,30 @@ def run(self, evidence, result):
     else:
       bulk_extractor_args = None
 
+    if self.task_config.get('regex_pattern_files'):
+      regex_pattern_files = self.task_config.get('regex_pattern_files')
+      result.log(f"Regex Files detected.")
+    else:
+      regex_pattern_files = None
+
     try:
       # Generate the command we want to run then execute.
       cmd = ['bulk_extractor']
 
+      # if evidence type is Directory we need to add an -R parameter to the command
+      if evidence.type == "Directory" or evidence.type == "CompressedDirectory":
+        result.log(
+            f"Running Bulk Extractor against {evidence.type:s} by using -R Flag."
+        )
+        cmd.extend(['-R'])
       cmd.extend(['-o', output_file_path])
 
       if bulk_extractor_args:
         cmd.extend(bulk_extractor_args)
+      if regex_pattern_files:
+        for regex_pattern_file in regex_pattern_files:
+          cmd.extend(['-F'])
+          cmd.extend([regex_pattern_file])
```

Comment:
you want to verify if these files actually exist?

Reply:
Sure, I will make this more robust.